src/formattedcode/writers.py

```diff
@@ -31,16 +31,20 @@
 import os
 from os.path import abspath
 
+import pluggy
+
 from formattedcode.format import as_template
 from formattedcode.format import as_html_app
 from formattedcode.format import create_html_app_assets
 from formattedcode.format import HtmlAppAssetCopyWarning
 from formattedcode.format import HtmlAppAssetCopyError
 
+from plugincode import hookspec
+
 
 def write_formatted_output(
         scanners, files_count, version, notice, scanned_files,
-        format, options, input, output_file, _echo):
+        format, options, input, output_file, _echo, plugin_formats, pm):
```

Comment:
It's the `post_scan` pluginmanager instance passed here by `cli.py`, I do get your point though (I hope), I'll rename it to be same as it is in `cli.py` i.e `post_scan_pm`